go.mod

```diff
@@ -28,4 +28,4 @@ require (
 	k8s.io/klog/v2 v2.2.0
 )
 
-go 1.14
+go 1.15
```

Comment:
This is not needed as we are not using anything specific to go 1.15 so our users don't require to switch to 1.15, so should be compatible with 1.14.